src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
```

Comment:
I think it's redundant
```suggestion

    public String getColor() {
        return color;
    }

    public int getNumber() {
        return number;
    }

```